lcgeoTests/CMakeLists.txt

```
@@ -84,6 +84,15 @@ ADD_TEST( t_${test_name} "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh
 SET_TESTS_PROPERTIES( t_${test_name} PROPERTIES FAIL_REGULAR_EXPRESSION  "Exception;EXCEPTION;ERROR;Error" )
 set_tests_properties( t_${test_name} PROPERTIES TIMEOUT 240) 
 
+#--------------------------------------------------
+# test for FCCee CAD beam pipe
+
+SET( test_name "test_FCCee_CAD_beampipe" )
+ADD_TEST( t_${test_name} "${CMAKE_INSTALL_PREFIX}/bin/run_test_${PackageName}.sh"
+	ddsim --compactFile=${CMAKE_CURRENT_SOURCE_DIR}/../FCCee/CAD_beampipe/compact/testBeampipe_CADimport_o1_v01.xml --runType=batch -G -N=1 --outputFile=testBeampipe_CADimport_o1_v01.slcio )
+SET_TESTS_PROPERTIES( t_${test_name} PROPERTIES FAIL_REGULAR_EXPRESSION  "Exception;EXCEPTION;ERROR;Error" )
```

```suggestion
SET_TESTS_PROPERTIES( t_${test_name} PROPERTIES FAIL_REGULAR_EXPRESSION  "ERROR;Error" )
```
Could you please remove `Exception;EXCEPTION` from the list of Regular expresion? 
3 facets are too small, and probably trigger this error when closing the geometry
```
14: -------- WWWW ------- G4Exception-START -------- WWWW -------
14: *** G4Exception : GeomSolids1001
14:       issued by : G4TessellatedSolid::SetSolidClosed()
14: Defects in solid: ASCII_shape_0x30e3ac0 - some facets have wrong orientation!
14: *** This is just a warning message. ***
14: -------- WWWW -------- G4Exception-END --------- WWWW -------
14: 
14: 
14: -------- WWWW ------- G4Exception-START -------- WWWW -------
14: *** G4Exception : GeomSolids1001
14:       issued by : G4TessellatedSolid::SetSolidClosed()
14: Defects in solid: ASCII_shape_0x30e3ac0 - there are holes in the surface!
14: *** This is just a warning message. ***
14: -------- WWWW -------- G4Exception-END --------- WWWW -------
```